### PR TITLE
[docs] Limitation to access secure registry images

### PR DESCRIPTION
**Pull request by mansikulkarni96**

### CODE REVIEW

**Comment by mansikulkarni96** [comment]
/ cc: @mburke5678

File: README.md [hunk 1]
```diff
@@ -228,6 +228,9 @@ deploy workloads.
 ### Cluster-wide proxy 
 WMCO does not support adding Windows workloads using a [cluster-wide proxy](https://docs.openshift.com/container-platform/latest/networking/enable-cluster-wide-proxy.html) config for the OpenShift Container Platform. WMCO will not be able to automatically route proxy connections for Windows workloads.
 
+### Accessing secure registries
+WMCO does not support pulling images from secure private registries on the Windows nodes. It is recommended to use images from public registries.
```
Comment (sebsoto) [inline README.md:232]: Alternatively the images can pre-pulled onto the VM image that is used

Comment (sebsoto) [reply README.md:232]: Or pulled before adding a BYOH instance to the cluster.

Comment (aravindhp) [reply README.md:232]: I would leave out the automation bit. That is up to the user.

File: README.md [hunk 2]
```diff
@@ -228,6 +228,9 @@ deploy workloads.
 ### Cluster-wide proxy 
 WMCO does not support adding Windows workloads using a [cluster-wide proxy](https://docs.openshift.com/container-platform/latest/networking/enable-cluster-wide-proxy.html) config for the OpenShift Container Platform. WMCO will not be able to automatically route proxy connections for Windows workloads.
 
+### Accessing secure registries
+WMCO does not support pulling images from secure private registries on the Windows nodes. It is recommended to use images from public registries or pre-pull the images in the VM golden image.
```
Comment (saifshaikh48) [inline README.md:232]: I'd drop the word golden since this is the general readme, not vsphere specific

Comment (saifshaikh48) [reply README.md:232]: also nit: line length

Comment (jrvaldes) [reply README.md:232]: +1

File: README.md [hunk 3]
```diff
@@ -228,6 +228,10 @@ deploy workloads.
 ### Cluster-wide proxy 
 WMCO does not support adding Windows workloads using a [cluster-wide proxy](https://docs.openshift.com/container-platform/latest/networking/enable-cluster-wide-proxy.html) config for the OpenShift Container Platform. WMCO will not be able to automatically route proxy connections for Windows workloads.
 
+### Accessing secure registries
+WMCO does not support pulling images from secure private registries on the Windows nodes.
```
Comment (sebsoto) [inline README.md:232]: This sentence could be restructured like so:
`Windows nodes managed by WMCO do not support pulling container images from secure private registries.`

I'd do this as the subject is the `Windows nodes managed by WMCO`, as WMCO itself is not doing the pulling.
I also replaced `images` with `container images`, to be explicit in differentiating it from the VM image referred to in the next sentence.

**Review by sebsoto — CHANGES_REQUESTED** [review]
Sorry, one more comment :sweat_smile:

File: README.md (hunk 3 above)
Comment (mansikulkarni96) [reply README.md]: Makes sense, thanks for the suggested wording. Made the changes.

File: README.md (hunk 1 above)
Comment (mburke5678) [reply README.md]: @mansikulkarni96 This is related to containerd, for 6.0.0+ only, correct?

Comment (mansikulkarni96) [reply README.md]: @mburke5678 I believe this was always the case, even with Docker. It's only with a feature request that came in with containerd that we decided to document it.

**Pull request merged**